Makefile

```diff
@@ -73,18 +75,18 @@ bin/proxy-server: bin cmd/proxy/main.go proto/agent/agent.pb.go proto/proxy.pb.g
 ## --------------------------------------
 
 .PHONY: gen
-gen: proto/agent/agent.pb.go proto/proxy.pb.go mock_gen
+gen: proto/agent/agent.pb.go proto/client/client.pb.go mock_gen
+
+proto/client/client.pb.go: konnectivity-client/proto/client/client.proto
+	protoc -I . konnectivity-client/proto/client/client.proto --go_out=plugins=grpc:.
+	cat hack/go-license-header.txt konnectivity-client/proto/client/client.pb.go > konnectivity-client/proto/client/client.licensed.go
+	mv konnectivity-client/proto/client/client.licensed.go konnectivity-client/proto/client/client.pb.go
 
 proto/agent/agent.pb.go: proto/agent/agent.proto
-	protoc -I proto proto/agent/agent.proto --go_out=plugins=grpc:proto
+	protoc -I ${GOPATH}/src sigs.k8s.io/apiserver-network-proxy/proto/agent/agent.proto --go_out=plugins=grpc:${GOPATH}/src
```

Comment:
See if you like this better: https://github.com/caesarxuchao/apiserver-network-proxy/commit/47d2b878dc877d87f2fd9992c5da0b3d3056a44c

I can't avoid using "GOPATH" either.

Reply:
Adopted your changes. :smile: